samples/Azure.ResourceManager.Sample/Generated/Models/VMScaleSetConvertToSinglePlacementGroupContent.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace Azure.ResourceManager.Sample.Models
 {
-    /// <summary> The VMScaleSetConvertToSinglePlacementGroupInput. </summary>
+    /// <summary> The VMScaleSetConvertToSinglePlacementGroupContent. </summary>
```

Comment:
I think this is a good impact of this change to make the description align with the class name.